src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const { type, extraData, keysToRemove } = action;
+
+    switch (true) {
+      case type === 'addProperties' :
```

Comment:
```suggestion
    switch (type) {
      case 'addProperties' :
```
In all cases you check the type, so why do you need to switch true?